spec/unit/util/log/destinations_spec.rb

```diff
@@ -46,13 +46,15 @@
 
     describe "on POSIX systems", :if => Puppet.features.posix? do
       describe "with a normal file" do
+        let (:parent) { Pathname.new('/tmp') }
         let (:abspath) { '/tmp/log' }
         let (:relpath) { 'log' }
 
         it_behaves_like "file destination"
 
         it "logs an error if it can't chown the file owner & group" do
-          expect(File).to receive(:exists?).with(abspath).and_return(false)
+          allow(File).to receive(:exist?).with(parent).and_return(true)
```

Comment:
Was this test giving false positives before?

Reply:
Ah earlier in the same method, we're using `Puppet::FileSystem.exist?` to check the parent directory, so I changed this one to match. I updated the commit message.